clinica/iotools/converters/oasis3_to_bids/oasis3_to_bids.py

```diff
@@ -57,16 +57,35 @@ def convert_clinical_data(self, clinical_data_dir, bids_dir):
                 else:
                     sessions_dict[y][z]["diagnosis"] = "CN"
 
+        for bids_id in bids_ids:
+            bids_subj_folder = path.join(bids_dir, bids_id)
+            session_paths = glob(path.join(bids_subj_folder, "*"))
+            for s in session_paths:
+                # Get the session name in month differences
+                s_name = path.basename(s).replace("ses-", "")
+                diff_months = str(sessions_dict[bids_id][s_name]['diff_months'])
+                s_new_name = "M" + diff_months
```

Comment:
Better not to cast diff_months to string yet and do it when computing the session name.

```suggestion
                diff_months = sessions_dict[bids_id][s_name]['diff_months']
                s_new_name = f"M{diff_months:02d}"
```

Reply:
It works perfectly for me! Noob question: do you know how to accept multiple suggestions in a single one commit on this github interface? Or do I have to make a commit for each of your suggestions. Or you can do it if you want... your choice ;)